libraries/matrixui/src/main/kotlin/io/element/android/libraries/matrix/ui/components/AttachmentThumbnail.kt

```diff
@@ -89,11 +110,11 @@ fun AttachmentThumbnail(
                     )
                 }
                 AttachmentThumbnailType.Location -> {
-                    PinIcon(
-                        modifier = Modifier.fillMaxSize()
+                    Icon(
+                        resourceId = R.drawable.ic_september_location,
+                        contentDescription = info.textContent,
```

Comment:
Was this change intended? Is this based on any designs?

Reply:
Good spot, and I did not advertise it (sorry!). Rational is coherency, see my comment https://github.com/vector-im/element-x-android/pull/1774#discussion_r1387642675

Reply:
I will revert the change to unblock the PR, since I would like the fix to be in the next release.